ui/app/settings.js

```diff
@@ -372,7 +373,7 @@ class Settings extends Component {
             h('div.settings__info-item', [
               h(
                 'div.settings__info-about',
-                'MetaMask is designed and built in California.'
+                t('californiaRoll')
```

Comment:
😆 nice, but will change to `builtInCalifornia`

Reply:
haha